daq/faucet_states_collector.py

```diff
@@ -131,8 +210,11 @@ def process_port_learn(self, timestamp, name, port, mac, src_ip):
             .setdefault(mac, {})
 
         global_mac_table[KEY_MAC_LEARNING_IP] = src_ip
-        global_mac_table[KEY_MAC_LEARNING_PORT] = port
-        global_mac_table[KEY_MAC_LEARNING_TS] = datetime.fromtimestamp(timestamp).isoformat()
+
+        global_mac_switch_table = \
```

Comment:
line breaks are very ugly, and generally always much better served by some other way. If it's over 100 chars, then I'd learn towards crating local intermediate variables, e.g

```
mac_learning_switch = global_mac_table.setdefault(KEY_MAC_LEARNING_SWITCH, {})
global_mac_switch_table = mac_learning_switch.setdefault(name, {})

```
same number of lines, but generally way more readable

Reply:
Done